src/pages/index.js

```diff
@@ -9,13 +9,10 @@ const IndexPage = () => {
     <Layout pageTitle="Home Page">
       <p>I'm making this by following the Gatsby Tutorial.</p>
 
-      <StaticImage
-        alt="Clifford, a reddish-brown pitbull, posing on a couch and looking stoically at the camera"
-        src="../images/icon.png"
-      />
+      <StaticImage alt="Gatsby" src="../images/icon.png" />
```

Comment:
One of the most important concepts at Elyk is website accessibility, so make sure your Alt text matches your images

Reply:
I usually do, in this case it was overlooked, Usually issues like this are found and fixed on code reviews which I usually do before publishing updates to a live site. This practice stemmed from working on raw html/css/js sites that have no CMS or back-end (static sites)

Reply:
Understood. I figured it was just something left-over from the tutorial and that you'd have fixed it. But A11y is important enough that I can't help but drill the info in whenever possible 🙂